app/ember/package.json

```diff
@@ -28,12 +28,15 @@
     "@babel/runtime": "^7.0.0",
     "@ember/test-helpers": "^0.7.25",
     "@storybook/core": "4.0.0-alpha.24",
+    "common-tags": "^1.8.0",
```

Comment:
Why is this moved?

Reply:
It is used in @storybook/ember and that's it.

Reply:
Doesn't ember itself depend on this dep?

Reply:
no, it is used for outputting error information in the render function